.gitignore

```diff
@@ -7,5 +7,6 @@
 /bin/
 /composer.phar
 /app/config/parameters.yml
+*.mo
```

Comment:
Overall, the code seems reasonable, and if it seems to be working, then merge it...

One thing that I'm not clear on... this commits the `*.pot` and `*.po` files and omits the `*.mo` files. I guess that reduces potential git-noise - so that's reasonable. But `*.mo` files are necessary, right? And there's nothing in the PR that talks about the compilation?

Perhaps the `README` should have some deployments notes about the `*.po`=>`*.mo` step?

Reply:
ok, I'll add a note in the README.

The .mo files are binary files, and can significantly grow the git repo. I prefer to avoid committing them.

Reply:
> The .mo files are binary files, and can significantly grow the git repo. I prefer to avoid committing them.

Big :+1: